### PR TITLE
Combined permission request with newer livekit sdk version

### PR DESCRIPTION
**Pull request by toger5**
requires: https://github.com/livekit/client-sdk-js/pull/773

This superseeds: https://github.com/vector-im/element-call/pull/1172

### CODE REVIEW

**Review by SimonBrandner — APPROVED** [review]
Looks sane though I have to admit it's a little hard keeping track of all things

File: package.json [hunk 1]
```diff
@@ -130,5 +130,8 @@
       "text",
       "cobertura"
     ]
+  },
+  "resolutions": {
+    "@types/react": "^18.2.12"
   }
```
Comment (toger5) [inline package.json]: This would fix the lint issues that were present in the build before this change. Is that acceptable?

Comment (SimonBrandner) [reply package.json]: Yeah, I think so, does anyone else from @vector-im/element-call-reviewers have opinions?

Comment (toger5) [reply package.json]: it was enought to update the yarn lock file

**Review by dbkr — APPROVED** [review]
Whoo, thanks!

File: src/room/VideoPreview.tsx [hunk 2]
```diff
@@ -149,25 +172,21 @@ export function VideoPreview({ matrixInfo, onUserChoicesChanged }: Props) {
...
-            />
-          )}
+          <MicButton
+            muted={!audioEnabled}
+            onPress={() => setAudioEnabled(!audioEnabled)}
+          />
+          <VideoButton
+            muted={!videoEnabled}
+            onPress={() => setVideoEnabled(!videoEnabled)}
+          />
```
Comment (daniel-abramov) [inline src/room/VideoPreview.tsx:182]: Just a quick question: what if we only have a video track or audio track?

Comment (toger5) [reply src/room/VideoPreview.tsx:182]: You are right. I did not like the jumpy behaviour of the buttons (popping in and out after the track is available.) That is why i disabled it.
This is addressed in: https://github.com/vector-im/element-call/pull/1218

**Pull request merged**